thecourseforum/theCourseForum2
# Fix uncaught TypeError in Developer Console

## Status
- Done

## GitHub Issues addressed (`#IssueNumber`)
- This PR closes #420.

## What I did
- Removed an unnecessary line to suppress an error on Developer Console

## Screenshots
|Before|After|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/7676354/110063802-d67c2480-7d39-11eb-9218-80cfdffd3bd6.png)|![image](https://user-images.githubusercontent.com/7676354/110064403-0972e800-7d3b-11eb-841d-9ec58f69f881.png)|

## Tests
1. Go to the landing page
2. Open developer console (Press F12 and go to the `Console` tab on Chromium-based browsers?)
3. You shouldn't see `Uncaught TypeError: Cannot read property 'addEventListener' of null`.
4. You should still be able to submit feedback on the landing page.

## Questions/Discussions/Notes
- N/A

## Merging the PR
- Who should merge this PR?
  - Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
